.github/CODEOWNERS

```diff
@@ -330,6 +332,8 @@
 
 /modules/services/xscreensaver.nix                    @rycee
 
+/modules/services/xsettingsd.nix                      @IvanMalison
+
```

Comment:
Seems like an extraneous change.

Reply:
Yeah, just forgot to do it in the last pr...

Reply:
I think this still needs to be removed.